### PR TITLE
PS-7024 Release Notes for 5.6.48-88.0

### PR DESCRIPTION
**Pull request by patrickbirch**
updated conf.py
updated release-notes_index
added Percona-Server-5.6.48-88.0

### CODE REVIEW

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.6.48-88.0.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6953`: If the LOAD DATA process found a line with an escape character followed by UTF8 characters, then the process failed to load the line and any additional lines. This issue was a regression. (Upstream :mysqlbug:`98089`)
+* :jirabug:`PS-6945`: The tokubackup process exported an incorrect API. This API mismatch prevented large file backups.
+* :jirabug:`PS-6946`: The address and thread sanitizers found memory use after free and other issues in the tokubackup software..
+* :jirabug:`PS-7020`: In Ubuntu 20.04, only python3-mysqldb is available. We converted the MTR tests from python2 to python3.
```
Comment (inikep) [inline doc/source/release-notes/Percona-Server-5.6.48-88.0.rst]: We converted python2 scripts to support both python2 and python3. It requires python2.6 or newer.

**Comment by inikep** [comment]
Please also add https://jira.percona.com/browse/PS-6966

File: doc/source/release-notes/Percona-Server-5.6.48-88.0.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-5.6.48-88.0.rst]: Fixed the "to" and added the sentence about Python2.6.

File: doc/source/release-notes/release-notes_index.rst [hunk 2]
```diff
@@ -6,6 +6,7 @@
    :maxdepth: 1
    :glob:
 
+   Percona-Server-5.6.48-88.0
```
Comment (percona-ysorokin) [inline doc/source/release-notes/release-notes_index.rst:9]: Just out of curiosity, why there is no 5.6.47 in this list?

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Review by inikep — APPROVED** [review]

File: doc/source/release-notes/release-notes_index.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/release-notes_index.rst:9]: GCA

Comment (inikep) [reply doc/source/release-notes/release-notes_index.rst:9]: Do we need GCA for docs? We always skip `doc` directory during null-merges.

**Pull request merged**